tests/phpunit/integration/Modules/AdSenseTest.php

```diff
@@ -171,6 +171,58 @@ public function test_block_on_consent_non_amp( $enabled ) {
 		}
 	}
 
+	/**
+	 * @dataProvider module_is_connected
+	 * @param bool $connected
+	 */
+	public function test_adsense_platform_tags( $connected ) {
```

Comment:
@kuasha420 connection status doesn't matter for platform tags. Let's get rid of it.

Reply:
Done. :)